policyengine_us/parameters/gov/states/ny/tax/income/supplemental/incremental_benefit/head_of_household.yaml

```diff
@@ -0,0 +1,26 @@
+description: NY supplemental tax incremental benefit amount for a head of household filer. 
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label: NY supplemental tax incremental benefit amount for head of household filers
+  reference:
+    - title: Section 601 Imposition of tax, (d-2) for 2021 and (d-3) for 2022
```

Comment:
is there a tax form reference as well (2021 and 2022)? - all yaml files

Reply:
2021 added. Let me know if you would like me to check the calculations for 2022 as the NY main income tax rates will need to be changed as well.

Reply:
That would be great! Could you add the 2022 rates to the parameters here as well